src/odemis/gui/conf/data.py

```diff
@@ -761,6 +776,15 @@
             ("band", {  # from filter
                 "label": "Filter",
             }),
+            # overwrites config
+            ("polarization", {
+                "control_type": odemis.gui.CONTROL_COMBO,
+                "label": "Polarization",
+            }),
+            ("acquireAllPol", {
+                "control_type": odemis.gui.CONTROL_CHECK,
+                "label": "Record all possible polarizations: ",
```

Comment:
That's a little long as a label. I'd suggest to name it `All polarizations`, and add a `tooltip` describing more, like `Record all 6 polarization types`.